contents/blog/posthog-vs-hotjar.mdx

```diff
@@ -10,626 +10,259 @@ featuredImage: >-
-  <ComparisonRow column1={true} column2="Only in Scale plan" feature="Console logs" description="Debug user problems and app errors." />
-  <ComparisonRow column1={true} column2={false} feature="Performance monitoring" description="Monitor network activity and performance" />
-  <ComparisonRow column1="Alpha" column2={false} feature="iOS app replays" description="Record user sessions in iOS apps." />
-  <ComparisonRow column1="Planned Q1 2024" column2={false} feature="Android app replays" description="Record user sessions in Android apps." />
-  <ComparisonRow column1={true} column2={false} feature="Custom playlists" description="Create playlists of related replays." />
-  <ComparisonRow column1={true} column2={true} feature="Download replays" description="Save replay files offline for storage." />
+  <ComparisonRow column1={true} column2={true} feature="Web app replays" description="Watch sessions in your web app" />
+  <ComparisonRow column1="Beta" column2={false} feature="Android replays" description="Watch real users use your Android app" />
+  <ComparisonRow column1="Beta" column2={false} feature="iOS replays" description="Watch real users use your iOS app" />
+  <ComparisonRow column1={true} column2={true} feature="Console logs" description="Debug user problems and app errors" />
+  <ComparisonRow column1={true} column2={false} feature="Network performance" description="See sessions web vitals and requests" />
+  <ComparisonRow column1={true} column2={false} feature="Custom playlists" description="Create playlists of related replays" />
+  <ComparisonRow column1={false} column2={true} feature="Create highlights" description="Crop and save segments of recordings" />
+  <ComparisonRow column1={false} column2={true} feature="Frustration and engagement scores" description="Filter recordings by scores" />
+  <ComparisonRow column1={true} column2={true} feature="Record via URL or event" description="Trigger recordings based on URL or event" />
+  <ComparisonRow column1={true} column2={false} feature="Record via feature flag" description="Trigger recordings based on feature flag variables" />
+  <ComparisonRow column1={true} column2={true} feature="Download replays" description="Save replay files offline for storage" />
 </ComparisonTable>
 
-- **Recording retention:** Hotjar keeps all replays for 365 days. PostHog retains all replays for up to three months, but retains replays saved to a playlist forever. PostHog also enables you to download replays as a `.json` file for safe keeping.
-
-> **Further reading:** [All the best Hotjar alternatives and competitors](/blog/best-hotjar-alternatives)
+> **Note:** Our mobile session replay is free while it's in development, and we're actively working on session replay support for React Native and Flutter, too. See our [mobile replay docs](/docs/session-replay/mobile) for more info on development and how replay works on mobile apps.
```

Comment:
Is this true? I didn't see anything about this and just assumed it had the same pricing as normal replay